client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/style.scss

```diff
@@ -20,13 +20,31 @@
 	@extend .notification-settings-title-light;
 	position: absolute;
 	inset-block-start: 50px;
-
 }
 
 .notification-settings__content {
 	margin-block-start: -16px;
 }
 
+.notification-settings__content-heading-with-beta {
+	display: flex;
+	align-items: center;
+	gap: 5px;
+}
+
+.notification-settings__beta-tag {
+	background: #8c8f94;
```

Comment:
Could we please use variables instead of hex code?

<img width="201" alt="Screenshot 2023-05-22 at 11 02 27 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/dd0dd48a-76b9-49a4-96b9-90b5ff413bda">